mitodl/micromasters
# Added PercolateQuery model

#### What are the relevant tickets?
Part of #2510 

#### What's this PR do?
Adds `PercolateQuery` model which adds and removes itself from Elasticsearch when its contents change.

#### How should this be manually tested?
Create a `PercolateQuery` locally with `{"query": {}}` as `query` (or something else that has a valid syntax). Go to `http://localhost:9100/micromasters/.percolator/{your_id}` and verify that the entry has the same query that you entered.
